apps/Pycharm CE/install

```diff
@@ -2,7 +2,9 @@
 
 install_packages python3-pip python3-dev openjdk-11-jdk build-essential clang || exit 1
 
-wget https://download.jetbrains.com/python/pycharm-community-2021.3.tar.gz -O pycharm-community.tar.gz || error "Failed to download pycharm-community.tar.gz"
+version="2021.3.1"
```

Comment:
just a note, the update apps PR has been merged and will detect the "" around the version as being a different version number and it will update them if you implement an update script for pycharm

just remove them, they aren't necessary

Reply:
> just a note, the update apps PR has been merged and will detect the "" around the version as being a different version number and it will update them if you implement an update script for pycharm


https://github.com/Botspot/pi-apps/pull/1371 @theofficialgman